neuro_data/pipeline_management/NeuroDataPipelineManagement.py

```diff
@@ -0,0 +1,139 @@
+import numpy as np
+import datajoint as dj
+
+
+from neuro_data.static_images.data_schemas import StaticScanCandidate, StaticScan, ConditionTier, Frame, InputResponse, Eye, Treadmill, StaticMultiDataset, StaticMultiDatasetGroupAssignment
+
+
+class NeuroDataPipelineManagement():
+    def __init__(self):
+        self.pipeline_anatomy = dj.create_virtual_module('pipeline_anatomy', 'pipeline_anatomy')
+        self.pipeline_fuse = dj.create_virtual_module('pipeline_fuse', 'pipeline_fuse')
+        self.pipeline_stimulus = dj.create_virtual_module('pipeline_stimulus', 'pipeline_stimulus')
+
+    def manually_insert_area_for_scan(self, target_scan, area):
+        neuron_unit_keys = (self.pipeline_fuse.ScanSet().Unit() & target_scan).fetch('KEY')
```

Comment:
newer version of datajoint (i think from 0.10 and up) don't need to call the instance of the class so this could be `self.pipeline_fuse.ScanSet.Unit & target_scan` which looks slightly nicer

Reply:
same for inserts and fetches

Reply:
I was aware of it. I just coded it that way in case someone was running the old datajoint, also force of habit. Does it really need to be change? I think it is okay as it is. Sure it does look a little clearer but not much.

Reply:
why not change it?

Reply:
Cause I have to go back and remove all of those ()

Reply:
Plus the code base in the other parts of this schema uses () when looking up tables.